modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp

```diff
@@ -62,6 +62,32 @@ EditBatchPlannerPlanDisplayContext editBatchPlannerPlanDisplayContext = (EditBat
 								/>
 							</div>
 
+							<div class="alert alert-autofit-stacked alert-indicator-start alert-info alert-inline">
+								<div class="alert-autofit-row autofit-row">
+									<div class="autofit-col autofit-col-expand">
+										<div class="autofit-section">
+											<span class="alert-indicator">
+												<svg class="lexicon-icon lexicon-icon-info-circle" focusable="false" role="presentation">
```

Comment:
`clay:icon` provides a wrapper for icons that makes the code more clean readable. Please use it